config/manifests/bases/awx-operator.clusterserviceversion.yaml

```diff
@@ -755,6 +755,10 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: Receptor Log Level
+        path: receptor_log_level
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
```

Comment:
@rooftopcellist i kind of guessed here. Is there an easy way for me to see how this gets rendered out?

Reply:
You can log in to the Openshift Console, install the AWX Operator from OperatorHub, then click into that operator under "Installed Operators" and click CSV Yaml, then you can add that x-decriptor to the CSV, save it and see it show up in the form.  

This is a handy way to quickly test out new changes to the form x-descriptors.